### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by juampynr**
userland > user and

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-26781--kubernetes-io-master-staging.netlify.app/docs/tasks/debug-application-cluster/debug-running-pod/#ephemeral-container-example

/kind cleanup

File: content/en/docs/tasks/debug-application-cluster/debug-running-pod.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ kubectl run ephemeral-demo --image=k8s.gcr.io/pause:3.1 --restart=Never
 ```
 
 The examples in this section use the `pause` container image because it does not
-contain userland debugging utilities, but this method works with all container
+contain user and debugging utilities, but this method works with all container
```
Comment (kbhawkey) [inline content/en/docs/tasks/debug-application-cluster/debug-running-pod.md:102]: 👋 @juampynr . Thanks for your contribution!
This change seems okay, though I am not clear on the meaning of "user and debugging utilities".
What do you think about changing the text to:
... contain debugging utilities, but this ...
or
... contain user debugging utilities, but this ...

Comment (juampynr) [reply content/en/docs/tasks/debug-application-cluster/debug-running-pod.md:102]: Hi @kbhawkey !

`contain debugging utilities, but this..` Sounds good to me. Changing...

Comment (kbhawkey) [reply content/en/docs/tasks/debug-application-cluster/debug-running-pod.md]: 👍

**Comment by kbhawkey** [comment]
Thanks @juampynr .
/lgtm

**Pull request merged**